applications/pelion_client/src/modules/pelion_fcc.cpp

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr.h>
+
+#include "factory_configurator_client.h"
+#include "pelion_fcc_err.h"
+
+#include <caf/events/button_event.h>
+
+#define MODULE pelion_fcc
+#include <caf/events/module_state_event.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(MODULE, CONFIG_PELION_CLIENT_PELION_LOG_LEVEL);
+
+
+static struct k_delayed_work fcc_init_work;
+static bool reset_storage;
+
+
+static void fcc_init_work_handler(struct k_work *work)
+{
+	fcc_status_e err = fcc_init();
+	if (err != FCC_STATUS_SUCCESS) {
+		LOG_ERR("Failed to initialize FCC (err:%s)", fcc_status_to_string(err));
+		module_set_state(MODULE_STATE_ERROR);
+		return;
+	}
+
+	if (reset_storage) {
+		LOG_WRN("Storage reset requested");
```

Comment:
Not LOG_INF?

Reply:
This is important - let's keep it visible